doc/advance_install.rst

```diff
@@ -0,0 +1,267 @@
+
+.. code-block:: bash
+
+  $ DEBUG=1 DJANGO_SETTINGS_MODULE=orga1.config python3 ./manage.py runserver
+
+You can then point your browser to `http://yourdomain.tld:8000` and check that you can see Pytitiont's home page and log-in with your newly created admin account.
+
+.. warning:: If you've set ``USE_MAIL_QUEUE`` to ``True`` and ``MAIL_EXTERNAL_CRON_SET`` to ``False``, running Pytition via ``manage.py runserver`` might not work well since you need to be run via `uwsgi`. Especially emails might not be sent.
+
+.. note:: If you switch ``USE_MAIL_QUEUE`` from ``False`` to ``True`` at some point, you might have to re-run ``python3 manage.py migrate`` to create the database structures needed for the mail queues.
+
+
+Apache + uwsgi + maintenance mode
+=================================
+
+Install uwsgi dependency::
+
+  sudo apt install uwsgi uwsgi-plugin-python3 python3-uwsgidecorators
+
+and enable proxy_uwsgi on apache:
+  sudo a2enmod proxy_uwsgi
+
+Here is an example of Apache configuration that you can put in `/etc/apache2/sites-available/orga1`::
+
+  <VirtualHost *:80>
+
+  ServerName pytition.orga1.org
+  
+  Alias /static /srv/pytition/www/static
+  Proxypass /static !
+  Alias /mediaroot /srv/pytition/www/mediaroot/orga1/
+  Proxypass /mediaroot !
+  
+  ProxyPass / unix:/var/run/uwsgi/app/pytition.orga1.org/socket|uwsgi://localhost/
+
+  <Directory /srv/pytition/www/static>
+  Require all granted
+  </Directory>
+  
+  <Directory /srv/pytition/www/mediaroot>
+  Require all granted
+  </Directory>
+  
+  CustomLog /var/log/apache2/access.log combined
+  CustomLog /var/log/apache2/pytition.orga1.org.log combined
+  
+  </VirtualHost>
+
+Voici un exemple de configuration pour uwsgi à mettre dans /etc/uwsgi/app-available avec un lien dans /etc/uwsgi/app-enabled
+
+  [uwsgi]
+  chdir = /srv/pytition/www/pytition/pytition
+  module = pytition.wsgi
+  home = /srv/pytition/pytition_venv
+  master = true
+  enable-threads = true
+  processes = 5
+  vacuum = true
+  socket = /var/run/uwsgi/app/pytition.orga1.org/socket
+  uid = orga1-user
+  gid = www-data
+  chmod-socket = 664
+  pythonpath = /etc/pytition/
+  plugins = python3
+  env = DJANGO_SETTINGS_MODULE=orga1.config
+  stats = 127.0.0.1:9191
+  need-app = true
+  max-requests = 5000                 
+  max-worker-lifetime = 3600
+  reload-on-rss = 2048
+  worker-reload-mercy = 60
+  harakiri = 120
+  py-callos-afterfork = true
+  auto-procname = true
+  procname-prefix = orga1->
+
+Start uwsgi and nginx servers:
+
+.. code-block:: bash
+
+  $ sudo systemctl start uwsgi
+  $ sudo systemctl start apache2
+
+Your Pytition home page should be available over there: http://pytition.orga1.org
+
+Now it's time to :ref:`Configure<Configuration>` your Pytition instance the way you want!
+
+Maintenance régulière (update)
+==============================
+Pour la mise à jour de tous les sites, voici un script (lancé par pytition-admin) qui peut être utilisé dans une tache cron:
```

Comment:
In order to update all your Pytition sites, here is a bach script (run by `pytition-admin` user) which can be used in a cron task: